core/client/controllers/posts/post.js

```diff
@@ -2,6 +2,20 @@ var PostController = Ember.Controller.extend({
     isPublished: Ember.computed.equal('model.status', 'published'),
     classNameBindings: ['model.featured'],
 
+    authorName: Ember.computed('model.author', function () {
+        return this.get('model.author.name') ? this.get('model.author.name') : this.get('model.author.email');
+    }),
```

Comment:
I'd make this:

```
authorName: Ember.computed('model.author.name', 'model.author.email', function () {
    return this.get('model.author.name') || this.get('model.author.email');
}),
```

That way the property will update when the name and email change, instead of only if the author object itself gets swapped out.